api/src/main/resources/marquez/db/migration/V40__alter_tables_resize_namespace_and_connection_url.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE dataset_versions ALTER COLUMN namespace_name TYPE VARCHAR;
+ALTER TABLE dataset_versions ALTER COLUMN dataset_name TYPE VARCHAR;
+ALTER TABLE datasets ALTER COLUMN name TYPE VARCHAR;
+ALTER TABLE datasets ALTER COLUMN physical_name TYPE VARCHAR;
+ALTER TABLE datasets ALTER COLUMN source_name TYPE VARCHAR;
+ALTER TABLE namespaces ALTER COLUMN name TYPE VARCHAR;
+ALTER TABLE runs ALTER COLUMN external_id TYPE VARCHAR;
+ALTER TABLE sources ALTER COLUMN name TYPE VARCHAR;
+ALTER TABLE sources ALTER COLUMN connection_url TYPE VARCHAR;
+ALTER TABLE tags ALTER COLUMN name TYPE VARCHAR;
```

Comment:
So these are to change from VARCHAR(256) to VARCHAR, right?

Reply:
yep